pkg/generators/models/testdata/cases/validation/expected/model_person_test.go

```diff
@@ -0,0 +1,20 @@
+package generatortest
+
+import (
+	"testing"
+
+	validation "github.com/go-ozzo/ozzo-validation/v4"
+	uuid "github.com/satori/go.uuid"
+	"github.com/stretchr/testify/require"
+)
+
+func TestPerson(t *testing.T) {
```

Comment:
where does this test come from? i couldn't find a template for it

Reply:
Oh, that's a leftover. I used it to check if the generated stuff compiles